pcsx2/IopBios.cpp

```diff
@@ -624,10 +626,12 @@ namespace R3000A
 		int dopen_HLE()
 		{
 			IOManDir* dir = NULL;
-			const std::string path = clean_path(Ra0);
+			std::string path = clean_path(Ra0);
 
 			if (is_host(path))
 			{
+				if(path == "host:/home/ty/Desktop/dopen-test/Picodrive/srm")
+					path += "/";
```

Comment:
I presume you meant to remove this :P

Reply:
Yeah my internet is dying and it wont let me push